.env.test

```
@@ -7,10 +7,4 @@ FLASK_DEBUG=true
 
 # Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
 SECRET_KEY=secret-key
-
-TRELLO_API_KEY=test-api-key
-TRELLO_TOKEN=test-token
-TRELLO_BOARD_ID=test-board-id
-TO_DO_LIST_NAME=test-todo
-DONE_LIST_NAME=test-done
-IN_PROGRESS_LIST_NAME=test-inprogress
\ No newline at end of file
+MONGODB_PRIMARY_CONNECTION="mongodb://fakemongo.com"
```

Pretty minor, but I'd suggest removing the quotes here - although `python-dotenv` handles this exactly as you'd expect, if were to pass the file through to docker at runtime with `--env-file .env.test` rather than copying it into the image, it actually treats the quotes as part of the literal string, which might break things:
```suggestion
MONGODB_PRIMARY_CONNECTION=mongodb://fakemongo.com
```